### PR TITLE
Adds tips/feedback section to docs

### PR DESCRIPTION
**Pull request by Etesam913**
## Overview
This pr adds a new section to the 'developing-frontends' page that goes over how to add tips/feedback to your react code and how to review tips/feedback using python scripts.

## Video of the updated section

https://user-images.githubusercontent.com/55665282/180496994-430a0b83-3b86-4a7b-af5b-46e0d9653274.mov

### CODE REVIEW

**Comment by Etesam913** [comment]
I think this makes sense, but an idea may be to put this in a separate page

**Comment by Etesam913** [comment]
Updated info on running review scripts:
Screen Shot 2022-07-21 at 11 21 29 AM
Screen Shot 2022-07-21 at 11 21 36 AM
Screen Shot 2022-07-21 at 11 21 40 AM

**Comment by pringshia** [comment]
Woohoo! Up-to-date docs 😍

I agree that I think this page might be getting too long, and might intimidate new users who come here to learn just about "Creating a task" (I know this is within the in-depth section, but still).

I think at the same level as "Worker quality control" we can introduce a page called "Worker experience" and shift this there instead?

We can also add a link to the new location in "Creating a task" so we can make sure we promote this feature.

File: docs/web/docs/reference/overview.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Other sections of the documentation may soon be auto-generated.
 In the meanwhile, you may find reference documentation in the following locations:
 
 - [`mephisto-task`](https://github.com/facebookresearch/Mephisto/tree/main/packages/mephisto-task) - This package provides two hooks to faciliate React-based front-end development for Mephisto tasks. Use `useMephistoTask` for simple, static tasks or `useMephistoLiveTask` for multi-turn, socket-based tasks.
+- [`mephisto-worker-addons`](https://github.com/facebookresearch/Mephisto/tree/main/packages/mephisto-worker-addons) - This package provides the tips and feedback components to allow for collection of tips and feedback from workers.
```
Comment (pringshia) [inline docs/web/docs/reference/overview.md:12]: good update

File: packages/mephisto-task/README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ function MyApp() {
 
 ## Documentation
 
-The `useMephisoTask` React hook exposes the following fields:
+The `useMephistoTask` React hook exposes the following fields:
```
Comment (pringshia) [inline packages/mephisto-task/README.md:60]: good catch!

File: docs/web/docs/guides/how_to_use/task_creation/developing_frontends.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ npm install mephisto-task
 
 The `mephisto-task` project surfaces three React hooks depending on your use case:
 
-1. `useMephistoTask` - Used for static tasks where one-time initial data is enough to power the task. See the example task in `/examples/static_react_task/` for an example project using this hook.
+1. `useMephistoTask` - Used for static tasks where one-time initial data is enough to power the task. See the example task in `/examples/static_react_task/` for an example project using this hook.
```
[the remark this answers is not in the document]
Comment (pringshia) [reply docs/web/docs/guides/how_to_use/task_creation/developing_frontends.md:19]: nbd really, but it overcomplicates things like `git blame`s

File: docs/web/docs/guides/how_to_use/task_creation/developing_frontends.md [hunk 4]
```diff
@@ -36,6 +36,186 @@ These suite of packages were formerly published under the `annotation-toolkit` a
 
 For chat-based components, we provide custom UI components in the `bootstrap-chat` package. You can find further information for them in the [associated README for the `bootstrap-chat` package](https://github.com/facebookresearch/Mephisto/tree/main/packages/bootstrap-chat).
 
+## Adding tips/feedback to tasks
+
+To allow for greater communication between workers to workers and it is recommended to use the Tips component. 
+
+### How to add Tips:
+1. Add the Tips component to your task's react webapp code
```
[the remark this answers is not in the document]
Comment (Etesam913) [reply docs/web/docs/guides/how_to_use/task_creation/developing_frontends.md:44]: Oh yeah, that's a big thing to forget to include 😅

**Comment by Etesam913** [comment]
> Woohoo! Up-to-date docs 😍
> 
> I agree that I think this page might be getting too long, and might intimidate new users who come here to learn just about "Creating a task" (I know this is within the in-depth section, but still).
> 
> I think at the same level as "Worker quality control" we can introduce a page called "Worker experience" and shift this there instead?
> 
> We can also add a link to the new location in "Creating a task" so we can make sure we promote this feature.

I updated the video to show these changed

**Review by JackUrb — APPROVED** [review]
Awesome to have the complete docs for this project available! Should be much easier for people to onboard now.

**Review by Rebecca-Qian — APPROVED** [review]
Looks really great! Thanks for adding documentation 🚀

**Pull request merged**